api/src/Microsoft.Azure.IIoT.OpcUa.Api.Publisher/src/Models/PublishNodesEndpointApiModel.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.Azure.IIoT.OpcUa.Api.Publisher.Models {
     /// A monitored and published item
     /// </summary>
     [DataContract]
-    public class PublishNodesRequestApiModel {
+    public class PublishNodesEndpointApiModel {
```

Comment:
@monikavar, I would still suggest to rename the class to `EndpointApiModel` as initial naming of `PublishNodesRequestApiModel` implied that it is used for `PublishNodes` direct method. That is no longer true as now we are using this class in the majority of the direct methods whether it is as a request or a response. So we can drop `PublishNodes` part of its name.

Reply:
`EndpointApiModel` already exists. Let's keep it as `PublishNodesEndpointApiModel` then.

Reply:
`EndpointApiModel` already exists so I would it as is.